javascript/javascript2/week1/shortestWord.js

```diff
@@ -0,0 +1,11 @@
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWord(array){
+    let shortWord = array[0]
+    for(let i = 0; i<array.length ; i++){
+        if (array[i].length < shortWord.length){
+            shortWord = array[i]
+        }
+    }
```

Comment:
You could assign the word to make this more readable:
```js
    for(let i = 0; i < array.length; i++){
        const currentWord = array[i];
        if (currentWord.length < shortWord.length){
            shortWord = currentWord
        }
    }
```

Reply:
I will fix that. I am really bad at write readable

Reply:
I don't think you're bad. But it's something that's good to keep in mind while programming.